azure-keyvault/azure/keyvault/secrets/_models.py

```diff
@@ -56,37 +56,37 @@ def __init__(self, **kwargs):
     @property
     def enabled(self):
         # type: () -> bool
-        """Gets the Secret's 'enabled' attribute"""
+        """Determines whether the object is enabled"""
         return self._management_attributes.enabled
 
     @property
     def not_before(self):
         # type: () -> datetime
-        """Gets the Secret's 'not_before' attribute"""
+        """Gets the Secret's 'not_before' date in UTC"""
```

Comment:
> """Gets the Secret's 'not_before' date in UTC""" [](start = 7, length = 49)

Personally I think we should leave the "Gets" out of these property descriptors, but it should be consistent for all the doc strings (vault_url, name and version don't say Gets)

Reply:
Removed the `Get's`, updated to use `The`.[here](https://github.com/samvaity/azure-sdk-for-python/blob/sv_secrets_docs_test/azure-keyvault/azure/keyvault/secrets/_models.py#L85)